athena-neptune/src/main/java/com/amazonaws/athena/connectors/neptune/NeptuneConnection.java

```diff
@@ -44,12 +44,12 @@
         cluster = builder.create();
     }
 
-    Client getNeptuneClientConnection()
+    public Client getNeptuneClientConnection()
     {
         return cluster.connect();
     }
 
-    GraphTraversalSource getTraversalSource(Client client)
+    public GraphTraversalSource getTraversalSource(Client client)
```

Comment:
Why public? seems package private is fine for how it is being used. please revert unless you need it to be literally public.

Reply:
Need it to accessible here: https://github.com/abhishekpradeepmishra/aws-athena-query-federation/blob/athena-neptune-edgesupport/athena-neptune/src/main/java/com/amazonaws/athena/connectors/neptune/propertygraph/PropertyGraphHandler.java